packages/storybook/stories/components/form-field/form-field.stories.ts

```diff
@@ -0,0 +1,230 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { defineCustomElement } from '@ovhcloud/ods-components/dist/components/ods-form-field';
+import { html } from 'lit-html';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+import { CONTROL_CATEGORY, orderControls } from '../../control';
+
+defineCustomElement();
+
+const meta: Meta = {
+  title: 'ODS Components/Form/Form Field',
+  component: 'ods-form-field',
+};
+
+export default meta;
+
+export const Demo: StoryObj = {
+  render: (arg) => html`
+  <ods-form-field error="${arg.error}">
+    ${unsafeHTML(arg.label)}
+    ${unsafeHTML(arg.visualHint)}
+    ${unsafeHTML(arg.mainSlot)}
+    ${unsafeHTML(arg.helper)}
+  </ods-form-field>
+    `,
+  argTypes: orderControls({
+    error: {
+      table: {
+        category: CONTROL_CATEGORY.general,
+        defaultValue: { summary: 'ø' },
+        type: { summary: 'string' }
+      },
+      control: 'text',
+    },
+    label: {
+      table: {
+        category: CONTROL_CATEGORY.slot,
+        defaultValue: { summary: 'ø' },
+      },
+      control: 'text',
+      description: 'Set a label on the top-left corner of the field.',
+    },
+    visualHint: {
+      table: {
+        category: CONTROL_CATEGORY.slot,
+        defaultValue: { summary: 'ø' },
+      },
+      control: 'text',
+      description: 'Set a visual hint on the top-right corner of the field.',
+    },
+    mainSlot: {
+      table: {
+        category: CONTROL_CATEGORY.slot,
+        defaultValue: { summary: 'ø' },
+      },
+      control: 'text',
+      description: 'Set the main field.',
+    },
+    helper: {
+      table: {
+        category: CONTROL_CATEGORY.slot,
+        defaultValue: { summary: 'ø' },
+      },
+      control: 'text',
+      description: 'Set a helper text under the field.',
+    },
+  }),
+  args: {
+    error: '',
+    label: `<ods-text slot="label" preset='label'>Description</ods-text>`,
+    visualHint: `<ods-text slot="visual-hint" preset='caption'>02/11/1999</ods-text>`,
+    mainSlot: `<ods-input type="text" value="Hello, ODS!" clearable></ods-input>`,
+    helper: `<ods-text slot="helper" preset='span'>A little helper text</ods-text>`,
+  },
+};
+
+export const Default: StoryObj = {
```

Comment:
Default would be the bare minimum required, in this case only input
Maybe use one of the complete one for the top doc page overview to have a better exemple and update this one?